source/less/blocks/site-navigation.less

```
@@ -0,0 +1,28 @@
+.site-navigation{
+  .center;
+}
+.site-navigation__list {
+  .li-none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.site-navigation__item{
+  border-top: 2px solid @border_bottem;
+}
+.site-navigation__link{
+  .a-none (@title_color);
+  .font__size (20px, 20px);
+  display: block;
+  width: 320px;
+  height: 66px;
+  font-weight: 500;
+  text-align: center;
+  padding-top: 22px;
+}
+.site-navigation____link:hover{
+  opacity: 0.6;
+}
+.site-navigation____link:active{
+  opacity: 0.3;
+}
```

```suggestion
.site-navigation__link:hover{
  opacity: 0.6;
}
.site-navigation__link:active{
  opacity: 0.3;
}
```

Лишние подчёркивания.